contrib/opencensus-samples/azure_monitor/README.rst

```diff
@@ -0,0 +1,63 @@
+Azure Monitor Samples
+=====================
+
+This package holds sample applications that demonstrate the usage of Azure Monitor exporters in OpenCensus.
+
+Installation
+------------
+
+Install OpenCensus Azure Monitor samples through `pip`.
+
+`pip install opencensus-azure-monitor-samples`
+
+
+Flask Sample
+------------
+
+The Azure Monitor Flask sample is a simple "To-Do" application.
```

Comment:
Why not having all description of the sample in a single paragraph?, the list format confused me a little bit